HTML/mockup-corporate/index.html

```diff
@@ -0,0 +1,329 @@
+                </picture>
+                <h3 class="section-heading section-heading--sub">Some awesome features</h3>
+                <p class="item__paragraph paragraph">
+                    Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit
+                    libero, a pharetra augue.
+                </p>
+            </div>
+            <div class="item">
+                <picture>
+                    <source media="(min-width: 1500px)" srcset="img/airplane@3x.png">
+                    <source media="(min-width: 500px)" srcset="img/airplane@2x.png">
+                    <img src="img/airplane.png" alt="plane icon" class="item__icon">
+                </picture>
+                <h3 class="section-heading section-heading--sub">Some awesome features</h3>
+                <p class="item__paragraph paragraph">
+                    Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit
+                    libero, a pharetra augue.
+                </p>
+            </div>
+            <div class="item">
+                <picture>
+                    <source media="(min-width: 1500px)" srcset="img/bulb@3x.png">
+                    <source media="(min-width: 500px)" srcset="img/bulb@2x.png">
+                    <img src="img/bulb.png" alt="bulb icon" class="item__icon">
+                </picture>
+                <h3 class="section-heading section-heading--sub">Some awesome features</h3>
+                <p class="item__paragraph paragraph">
+                    Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit
+                    libero, a pharetra augue.
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section class="container section-responsive">
+        <div class="section-responsive__trapeze">
+        </div>
+        <div class="section-responsive__icon">
+            <picture>
+                <source media="(min-width: 1500px)" srcset="img/macbook@3x.png">
+                <source media="(min-width: 500px)" srcset="img/macbook@2x.png">
+                <img src="img/macbook.png" alt="macbook icon">
+            </picture>
+        </div>
+        <div class="section-responsive__content">
+            <p class="section-heading section-heading--label">NEW DESIGN</p>
+            <h2 class="section-heading">
+                Resposive design, just need your tap <span>....</span>
+            </h2>
+            <p class="section-responsive__paragraph paragraph">
+                Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id
+                elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiasem malesuada magn.
+                Vestibulum felis euismod semper.
+            </p>
+        </div>
+    </section>
+
+    <section class="container section-people">
+        <div class="section-people__items">
+            <div class="item">
```

Comment:
what item, I understand that you use it in different places, but would be great to specify a bit, as every element on page can be an item